src/GitVersion.Core/Configuration/EffectiveBranchConfiguration.cs

```diff
@@ -7,4 +8,12 @@ public record EffectiveBranchConfiguration(EffectiveConfiguration Value, IBranch
     public IBranch Branch { get; } = Branch.NotNull();
 
     public EffectiveConfiguration Value { get; } = Value.NotNull();
+
+    public NextVersion CreateNextVersion(BaseVersion baseVersion, SemanticVersion incrementedVersion)
```

Comment:
This method was already moved to `NextVersionCalculator` where the usage is, please that one instead, I prefer to have this one POCO